sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectSummary.groovy

```diff
@@ -13,8 +13,8 @@ import life.qbic.datamodel.dtos.projectmanagement.Project
  */
 @EqualsAndHashCode
 class ProjectSummary {
-    String code
-    String title
+    private String code
+    private String title
```

Comment:
Why did you make them private?

Reply:
Because I would assume that code and title will not change but the sample numbers